### DISCUSSION TITLE
Required mail from institution but institution does not provide it

### DISCUSSION THREAD

**Discussion by byalexandrepedrosa** [community/community#60650] [GitHub Education]
### Hi everyone,

I lost my student pack saying cant continue because I'm not using email address from my University...
The graduation I'm doing is all online and theres no e-mail from their domain to students...

I am canceling my Office 365 subscription and informing Microsoft of my displeasure with this situation.

**Comment by OshekharO** [comment]
If you don't have mail you can use your transcript or fee receipt as a proof for education benefits, if still not approved try contacting GitHub supports.

**Reply by byalexandrepedrosa** [reply]
Already have sended Enrollment Proof and the automatic answer is always same:

Attendees of this school must verify and use their academic email address

**Comment by nonvenomous** [comment]
I'm in the same situation.
I attend a school that is kinda in between a school and university.
Today I went to the school secretary and received a certificate for school enrollment.
So I just tried reverifying my academic affiliation with the new information I have but I got auto-rejected the same minute.
 
![image](https://github.com/community/community/assets/37549874/6b815712-4a06-4e39-b0b9-99edfb84bd49)

My school doesn't provide email addresses to students, at least not to the 3 year programming study that I'm doing.

I'm not sure what I'm supposed to do now.

**Reply by byalexandrepedrosa** [reply]
I emailed github support the same day and so far I haven't gotten a response.

I have already canceled all automatic renewals for my Microsoft products and explained what happened here as a reason.

In the meantime, I'm looking at the competition, OneDrive, which I have the family plan, I'm going to replace it with owncloud or similar, github, if the non-renewal for students is definitive, I'll put gitlab and evangelize its use, and so on.

**Comment by kaifkh20** [comment]
Same thing happened with me today.

**Reply by byalexandrepedrosa** [reply]
Mail Microsoft too, seems github support does not answer the tickets.
